src/order-and-subscriptions/OrderAndSubscriptionsPage.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import Subscriptions from '../subscriptions';
+import Payments from '../order-history';
+
+import messages from './OrderAndSubscriptionsPage.messages';
+
+const OrderAndSubscriptionsPage = ({ intl }) => {
```

Comment:
Since it's a functional component prefer using `intl` object from `useIntl` hook.

```js
import { useIntl } from '@edx/frontend-platform/i18n';

const intl = useIntl();
```